ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -302,11 +302,57 @@ func convertToECSVolumes(hostPaths *adapter.Volumes) []*ecs.Volume {
 			}}
 		output = append(output, ecsVolume)
 	}
-	// volumes with an empty host path
-	for _, volName := range hostPaths.VolumeEmptyHost {
+
+	// process docker volumes
+	dockerVolumes := make(map[string]DockerVolume)
+	if ecsParams != nil {
+		for _, dockerVol := range ecsParams.TaskDefinition.DockerVolumes {
+			if dockerVol.Name != "" {
+				dockerVolumes[dockerVol.Name] = dockerVol
+			}
+		}
+	}
+
+	// volumes without host path (allowed to have Docker Volume Configuration)
+	output = append(output, mergeVolumesWithoutHost(hostPaths.VolumeEmptyHost, ecsParams)...)
+	return output
+}
+
+func mergeVolumesWithoutHost(composeVolumes []string, ecsParams *ECSParams) []*ecs.Volume {
+	volumesWithoutHost := make(map[string]DockerVolume)
+	output := []*ecs.Volume{}
+
+	for _, volName := range composeVolumes {
+		volumesWithoutHost[volName] = DockerVolume{}
+	}
+
+	if ecsParams != nil {
+		for _, dockerVol := range ecsParams.TaskDefinition.DockerVolumes {
+			if dockerVol.Name != "" {
+				log.Info("is it right here:")
+				log.Info(dockerVol)
```

Comment:
I'm guessing the 2 comment above were for debugging and should be removed?

Reply:
oops...